<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#section_title {
	font-size: 15pt;
    font-weight: bold;
	border: none;
	padding-bottom: 10px;
	border-bottom: 2px solid #eeeff3;
}

#manage_content {
	background-color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="section_title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Application Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ShadowFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="manage_content">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>809</width>
           <height>457</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font-size: 12pts;
font-weight: 500;</string>
              </property>
              <property name="text">
               <string>START SECTION SETTING</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">border: none;
border-top: 1px solid gray;
margin-top: 13px;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Start on</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="default_location_btn">
                <property name="text">
                 <string>System Default (Simulate Section)</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="custom_location_btn">
                <property name="text">
                 <string>Custom Location</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Choose Location</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="start_up_combobox">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Simulate Section</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Learning Section</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quiz Section</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Profile Section</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Settings Section</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="rightMargin">
               <number>30</number>
              </property>
              <item alignment="Qt::AlignRight">
               <widget class="QPushButton" name="reset_btn">
                <property name="text">
                 <string>Reset System</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">border: none;
border-top: 1px solid gray;
margin-top: 13px;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">font-size: 12pts;
font-weight: 500;</string>
              </property>
              <property name="text">
               <string>QUIZ SECTION SETTING</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Custom Quiz Setting</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="ToggleSwitch" name="expand_toggle">
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="custom_quiz_expand">
                <property name="text">
                 <string>Auto</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShadowFrame</class>
   <extends>QFrame</extends>
   <header>widgets/shadow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToggleSwitch</class>
   <extends>QCheckBox</extends>
   <header>widgets/clickables.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
